### DISCUSSION TITLE
ip SSRF improper categorization in isPublic in Bootstrap 5.3.3

### DISCUSSION THREAD

**Discussion by vierjulia** [twbs/bootstrap#40957] [General] [answered]
After I forked the repo, I `npm install` it then it showing some high severe vulnerability after the command `npm audit`

https://github.com/advisories/GHSA-2p57-rm9w-gvfp

### npm audit report

---

![image](https://github.com/user-attachments/assets/d86c0841-ffd7-4d18-aa55-85d6eb1f5a15)

ip  *
Severity: high
ip SSRF improper categorization in isPublic - https://github.com/advisories/GHSA-2p57-rm9w-gvfp
No fix available
node_modules/ip

1 high severity vulnerability

Some issues need review, and may require choosing
a different dependency.

**Comment by julien-deramond** [comment, marked as answer]
The `ip` package is only used for the Karma BrowserStack or debug unit tests, not in Bootstrap library itself. So nothing to worry about.
